src/hotkey/hotkey_command.cpp

```diff
@@ -115,7 +115,7 @@ std::array<hotkey_command_temp, HOTKEY_NULL - 1> master_hotkey_list {{
 	{ HOTKEY_CREATE_UNIT, "createunit", N_("Create Unit (Debug!)"), false, scope_game, HKCAT_DEBUG, "" },
 	{ HOTKEY_CHANGE_SIDE, "changeside", N_("Change Side (Debug!)"), false, scope_game, HKCAT_DEBUG, "" },
 	{ HOTKEY_KILL_UNIT, "killunit", N_("Kill Unit (Debug!)"), false, scope_game, HKCAT_DEBUG, "" },
-	{ HOTKEY_PREFERENCES, "preferences", N_("Preferences"), false, scope_game | scope_editor | scope_main, HKCAT_GENERAL, "" },
+	{ HOTKEY_PREFERENCES, "preferences", N_("Go to Preferences menu"), false, scope_game | scope_editor | scope_main, HKCAT_GENERAL, "" },
```

Comment:
Maybe "Open Preferences Screen"?

I think the 'm' in 'menu' should be a capital letter.